src/components/Login/Login.js

```diff
@@ -194,6 +207,20 @@ class Login extends React.Component {
               />
             </label>
             <br />
+            <label className="form-group">
```

Comment:
we need to add a constant "IS_ADMIN_VERSION" and set it to true and only show the email input if this is true ... so we can easily turn off the email input.